README.md

```diff
@@ -12,11 +12,13 @@ dynamodump supports local DynamoDB instances as well (tested with [dynalite](htt
 Usage
 -----
 ```
-usage: dynamodump.py [-h] [-a {zip,tar}] [-b BUCKET] [-m MODE] [-r REGION]
-                     [--host HOST] [--port PORT] [--accessKey ACCESSKEY]
-                     [--secretKey SECRETKEY] [-p PROFILE] [-s SRCTABLE]
-                     [-d DESTTABLE] [--prefixSeparator PREFIXSEPARATOR]
-                     [--noSeparator] [--readCapacity READCAPACITY] [-t TAG]
+usage: dynamodump.py [-h] [-a {zip,tar}] [-b BUCKET]
```

Comment:
This was autogenerated by running python dynamodump.py -h on python 2.7.12.

Let me know if you want me to revert back to the original format.